internal/bundle/yacit/main.go

```diff
@@ -17,7 +17,7 @@ import (
 func main() {
 	res := yacit.New(yacit.YacItConfig{
 		SkipUnmarshal: []string{
-			"Impact", "ImpactValue", "Filters", "Remediation",
+			"Impact", "ImpactValue", "Filters", "Remediation", "RiskMagnitude",
```

Comment:
just curious: does having the field here means that it will be unmarshalled using our own `UnmarshalJSON` func?

Reply:
It means don't generate the unmarshal function because we will write it manually